codalab/rest/util.py

```diff
@@ -129,6 +129,8 @@ def get_bundle_infos(
         worksheet_names = _get_readable_worksheet_names(model, worksheet_uuids)
 
         for bundle_uuid, host_uuids in host_worksheets.items():
+            if bundle_uuid not in bundle_infos:
```

Comment:
I'm a little confused as to why `bundle_uuid` would not be in `bundle_infos`. Do you know when this is the case? We build `bundle_infos` from all the bundles we want to query and the keys of `host_worksheets` (on line 131) should only have bundle uuid's queried from the database itself (without any worksheet uuid's I believe).

Reply:
I think it's the query difference in get_host_worksheet_uuids and batch_get_bundles, if we query an invalid bundle uuid in batch_get_bundles, we get an empty list, and it returns here https://github.com/codalab/codalab-worksheets/blob/ada3fa9950136595a7226f2996d14c4d887c27f2/codalab/model/bundle_model.py#L685, maybe need to do some digging in the query